### PR TITLE
feat: Clarify the calculation of coverage values DOCS-523

### PR DESCRIPTION
**Pull request by nicklem**
Clarifies the calculation of coverage values displayed on the dashboard (and Codacy badge)

### :eyes: Live preview
https://feat-coverage-values-displayed-docs--docs-codacy.netlify.app/faq/code-analysis/which-metrics-does-codacy-calculate/#code-coverage

### CODE REVIEW

File: docs/faq/code-analysis/which-metrics-does-codacy-calculate.md
Comment (nicklem) [inline docs/faq/code-analysis/which-metrics-does-codacy-calculate.md:149]: @fjrdomingues 
Here's [our original discussion](https://codacy.slack.com/archives/C04CMEZT6ER/p1676471176146659?thread_ts=1676469139.213069&cid=C04CMEZT6ER) for context.

I'm also updating the description of the Codacy badge since it appears to show the same value (truncated coverage of the most recent commit). Let me know if this is not the case, thanks!

Comment (prcr) [reply docs/faq/code-analysis/which-metrics-does-codacy-calculate.md:149]: It's actually possible to have coverage badges [for other branches besides the default one](https://docs.codacy.com/getting-started/adding-a-codacy-badge/). So, I'd make the sentence about the badge more generic:

```suggestion
|[Repository Dashboard](../../repositories/repository-dashboard.md)|Coverage of the most recent commit of the selected branch and its evolution over time|
|[Codacy badge](../../getting-started/adding-a-codacy-badge.md)|Coverage of the most recent commit of the configured branch|
```

Comment (nicklem) [reply docs/faq/code-analysis/which-metrics-does-codacy-calculate.md:149]: Good catch, thanks!

**Review by prcr — APPROVED** [review]
This is a small change/fix, so we can merge it without waiting for @fjrdomingues' approval. :rocket:

**Pull request merged**
